ampl-converter/src/main/java/com/powsybl/ampl/converter/version/BasicAmplExporter.java

```diff
@@ -984,6 +999,12 @@ public void writeDanglingLineToFormatter(TableFormatter formatter, DanglingLine
         double p2 = sv.getP();
         double q2 = sv.getQ();
         double patl = getPermanentLimit(dl.getCurrentLimits().orElse(null));
+
+        if (busNum == middleBusNum) { // Not sure if this condition might occur
+            LOGGER.warn("Skipping dangling line '{}' connected to the same bus at both sides", id);
```

Comment:
A suggestion for the log, to emphasize the fact that it's the pairing key that is wrong
```suggestion
            LOGGER.warn("Skipping dangling line '{}': the pairing key refers to its bus.", id);
```